supply/lib/supply/options.rb

```diff
@@ -68,12 +68,19 @@ def self.available_options
                                      env_name: "SUPPLY_APK",
                                      description: "Path to the APK file to upload",
                                      short_option: "-b",
+                                     conflicting_options: [:apk_dir],
                                      default_value: Dir["*.apk"].last || Dir[File.join("app", "build", "outputs", "apk", "app-Release.apk")].last,
                                      optional: true,
                                      verify_block: proc do |value|
                                        UI.user_error! "Could not find apk file at path '#{value}'" unless File.exist?(value)
                                        UI.user_error! "apk file is not an apk" unless value.end_with?(value)
                                      end),
+        FastlaneCore::ConfigItem.new(key: :apk_dir,
```

Comment:
Another option is to make this param actually be an array of apk paths instead of a directory and you could use the `GRADLE_ALL_APK_OUTPUT_PATHS` output from the [gradle](https://github.com/fastlane/fastlane/blob/master/fastlane/lib/fastlane/actions/gradle.rb) action as the default!